pkg/imageverifier/bindir/bindir.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/containerd/containerd/v2/internal/tomlext"
```

Comment:
I think you also need https://github.com/DataDog/containerd/commit/285079c1ed4d3ce297435d7de8df0ac2a88f36b3

Reply:
It's in https://github.com/DataDog/containerd/pull/11

Reply:
oops sorry lgtm then!